### ISSUE TITLE
Mission Ignore List

### ISSUE THREAD

**Issue by Paixtaras** [NatesHonor/MissionchiefBot-X#6]
Great work at avoiding alliance missions.
A mission ignore list would be ideal, in my opinion right now.
Some missions expand and call for vehicles that some players haven't yet purchased.
It would be ideal if you could ignore these missions and refrain from scanning them, as the script sends available troops there and leaves none for other missions.

**Comment by NatesHonor** [comment]
Implemented - Will be in v2.0.1

**Comment by NatesHonor** [comment]
I have now applied the mission ignore list in commit [aa0e6f5](https://github.com/NatesHonor/MissionchiefBot-X/commit/aa0e6f517c433b9a04f529d96a2cd66aad2eab7c). Each region now has `data/mission_ignore_list.json`, supporting active mission IDs, exact mission names, and name fragments. The collector filters configured missions before detailed pages are scanned, releases their cached vehicle locks, and the behavior is covered by automated tests. This is targeted for MissionchiefBotX v3.1.0.

**Comment by NatesHonor** [comment]
I have verified and documented the Mission Ignore List implementation in [aa0e6f5](https://github.com/NatesHonor/MissionchiefBot-X/commit/aa0e6f517c433b9a04f529d96a2cd66aad2eab7c).

The list is configured per region at:

- `regions/us/data/mission_ignore_list.json` for the US service
- `regions/uk/data/mission_ignore_list.json` for the UK service
- `regions/<region>/data/mission_ignore_list.json` for every other supported region

It supports three matching methods:

- `mission_ids`: exact active mission IDs for one-off exclusions
- `mission_names`: exact mission names, normalized for accents and punctuation
- `contains`: name fragments for stable categories whose active IDs may change

Example:

```json
{
  "mission_ids": ["123456789"],
  "mission_names": ["Mission name to skip"],
  "contains": ["airport"]
}
```

The collector loads this file before opening detailed mission pages, removes matching missions from the dispatch set, and releases any cached vehicle reservations for them. Empty files keep all missions enabled. This is targeted for MissionchiefBotX v3.1.0.

**Comment by NatesHonor** [comment]
This issue was updated in commit [aa0e6f5](https://github.com/NatesHonor/MissionchiefBot-X/commit/aa0e6f517c433b9a04f529d96a2cd66aad2eab7c).
The fix will be included in MissionchiefBotX v3.1.0.
